linux_os/guide/system/auditing/auditd_configure_rules/directory_group_ownership_var_log_audit/tests/correct_value.pass.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+chgrp root /var/log/audit
```

Comment:
What about testing also the case when log_group in auditd.conf is not root?

Reply:
That's a good idea... I think so far we haven't tested this case in other rules that behave the same.